### PR TITLE
Mapper updates

### PR DESCRIPTION
**Pull request by JohnPadgettCivica**

### CODE REVIEW

File: src/EA.Weee.Web.Tests.Unit/Areas/Aatf/Controller/ManageEvidenceNotesControllerIndexTests.cs [hunk 1]
```diff
@@ -117,37 +119,72 @@ public async void IndexGet_GivenRequiredData_ModelMapperShouldBeCalled()
...
+
         [Fact]
         public async void IndexGet_GivenRequiredData_ModelShouldBeReturned()
         {
             //arrange
-            var organisationId = Guid.NewGuid();
-            var aatfId = Guid.NewGuid();
-            var model = new ManageEvidenceNoteViewModel();
-            A.CallTo(() => Mapper.Map<ManageEvidenceNoteViewModel>(A<ManageEvidenceNoteTransfer>._)).Returns(model);
+            var manageNoteViewModel = new ManageEvidenceNoteViewModel();
```
Comment (meglenapnovoselska) [inline src/EA.Weee.Web.Tests.Unit/Areas/Aatf/Controller/ManageEvidenceNotesControllerIndexTests.cs:156]: Fixture.Create<>() for those two?

File: src/EA.Weee.Web.Tests.Unit/Areas/Aatf/Controller/ManageEvidenceNotesControllerIndexTests.cs [hunk 2]
```diff
@@ -117,37 +119,72 @@ public async void IndexGet_GivenRequiredData_ModelMapperShouldBeCalled()
...
-        public async void IndexGet_GivenAction_DefaultViewShouldBeReturned()
+        public async void IndexGet_GivenAction_EditDraftReturnedNotesOverviewViewShouldBeReturned()
         {
             var organisationId = Guid.NewGuid();
             var aatfId = Guid.NewGuid();
 
             var result = await ManageEvidenceController.Index(organisationId, aatfId) as ViewResult;
 
-            result.ViewName.Should().BeEmpty();
+            result.ViewName.Should().Be("Overview/EditDraftReturnedNotesOverview");
```
Comment (meglenapnovoselska) [inline src/EA.Weee.Web.Tests.Unit/Areas/Aatf/Controller/ManageEvidenceNotesControllerIndexTests.cs:181]: worth keeping those in a constant and share them between the controller and tests?

File: src/EA.Weee.Web.Tests.Unit/Areas/Aatf/Controller/ManageEvidenceNotesControllerIndexTests.cs [hunk 3]
```diff
@@ -117,37 +119,72 @@ public async void IndexGet_GivenRequiredData_ModelMapperShouldBeCalled()
...
+
+            model.Should().Be(evidenceNoteViewModel);
+            model.ManageEvidenceNoteViewModel.Should().Be(manageNoteViewModel);
         }
 
         [Fact]
-        public async void IndexGet_GivenAction_DefaultViewShouldBeReturned()
+        public async void IndexGet_GivenAction_EditDraftReturnedNotesOverviewViewShouldBeReturned()
         {
             var organisationId = Guid.NewGuid();
```
Comment (meglenapnovoselska) [inline src/EA.Weee.Web.Tests.Unit/Areas/Aatf/Controller/ManageEvidenceNotesControllerIndexTests.cs:176]: could you add  act, assert, arrange ?

**Review by meglenapnovoselska — APPROVED** [review]

File: src/EA.Weee.Web.Tests.Unit/Areas/Aatf/Controller/ManageEvidenceNotesControllerIndexTests.cs (hunk 2 above)
Comment (JohnPadgettCivica) [reply src/EA.Weee.Web.Tests.Unit/Areas/Aatf/Controller/ManageEvidenceNotesControllerIndexTests.cs:181]: Sounds good

**Pull request merged**